README.md

```diff
@@ -26,16 +26,10 @@ cat ~/.flightctl/client.yaml | yq '.service.certificate-authority-data' | base64
 
 Choose one of the two options to run the UI application
 
-1. With auth disabled:
```

Comment:
Seems like we should remove the section about copying the backend certificate from the README too.
Unless it's necessary for the "ca_oidc" which I don't fully understand ATM.

Reply:
I've updated the README to correctly reflects what options we have regarding the setup